lib/bundler.js

```diff
@@ -169,12 +177,27 @@ module.exports = async function bundler({
             bundle.dependencies.forEach( dependency => loadBundle.push(dependency));
 
         } else if(Array.isArray(dependencies)){
-
             //dependency using extension bundles
             dependencies.forEach( dependency => loadBundle.push(`${dependency}/${outputDir}/${dependency}.min`));
         }
-        if(loadBundle.length){
 
+        // exclude dependencies from bundles:
+        if (Array.isArray(dependencies)) {
+            //dependency using extension bundles
+            dependencies.forEach( async (dependency) => {
```

Comment:
Be careful, because you won't wait for the resolve of these promises, so the execution will go forward before all `excludeModules` will be finished. You should use `Promise.all` as we discussed.

Reply:
`await Promise.all()` solution works :)